src/M6Web/Bundle/DaemonBundle/Tests/Units/Command/DaemonCommandConcreteIterationCallback.php

```diff
@@ -29,6 +29,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
     protected function myCallback(InputInterface $input, OutputInterface $output)
     {
-        $this->countCall++;
+        ++$this->countCall;
```

Comment:
Actually, this code has been updated when using [php-cs-fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer)  with the `@Symfony` rules set.
> pre_increment [@Symfony]
> Pre incrementation/decrementation should be used if possible.